typescript/ccip-server/src/services/common/Requestor.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosInstance } from 'axios';
```

Comment:
nit: use native node fetch

Reply:
Removed Requestor.ts and axios in favor of fetch.